2-mandatory/4-people-i-know.js

```diff
@@ -402,6 +429,18 @@ This time, I only want the full names of the people who can multitask
 */
 
 let colleaguesWhoCanMultitask = [];
+friends.forEach((friend) => {
+  const theirColleagues = friend.colleagues;
+
+  theirColleagues.forEach((colleague) => {
+    const colleagueSkills = colleague.skills;
+    const colleagueName = colleague.name;
+
+    if (colleagueSkills.includes("Multi-tasking")) {
```

Comment:
how to ensure that there are no repeating names in the ``colleaguesWhoCanMultitask`` array?